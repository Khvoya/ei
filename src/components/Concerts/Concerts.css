.Concerts {
  height: 720px;
  padding: 100px 11.7%;
  display: flex;
  flex-direction: row;
}
.Concerts__image,
.Concerts__content {
  width: 50%;
  height: 100%;
}

.Concerts__image {
  background-image: url("../../media/concerts_bg.png"),
    url("../../media/concerts-bg2.png");
  background-repeat: no-repeat;
  background-size: cover;
  display: flex;
  flex-direction: column;
  justify-content: center;
}
.Concerts__image span {
  color: #ffffff;
  font-family: Roboto;
  font-size: 60px;
  font-weight: 400;
  line-height: 73.79px;
}

.Concerts__content {
    width: 50%;
}
