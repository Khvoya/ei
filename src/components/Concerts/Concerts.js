import React, { Component } from "react";
import "./Concerts.css";
import { Container, Col, Row } from "react-bootstrap";
import { CONCERTS } from "../../constants/concerts";
import ConcertItem from "../../components/Concerts/ConcertItem/ConcertItem";

export const Concerts = () => (
  <div className="Concerts__container">
    <Container id="concerts" className="Concerts">
      <Row className="Concerts__image">
        <Col className="Concerts__title">
          <h1>Gigs</h1>
        </Col>
      </Row>
      <Row>
        <Col className="Concerts__text">
          <p>
            If you want to see us at your location, please contact us <br />
            <a href="mailto:electricindiansband@gmail.com">
              electricindiansband@gmail.com
            </a>
          </p>
        </Col>
      </Row>
      <Row>
        <Col className="Concerts__content">
          {CONCERTS.length > 0 &&
            CONCERTS.map(item => (
              <ConcertItem
                key={item.id}
                date={item.date}
                country={item.country}
                club={item.club}
              />
            ))}
        </Col>
      </Row>
    </Container>
  </div>
);

export default Concerts;
