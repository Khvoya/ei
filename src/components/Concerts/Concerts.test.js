import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Concerts from './Concerts';

describe('Concerts', () => {

  it('should render correctly', () => {
    const output = shallow(
      <Concerts />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
