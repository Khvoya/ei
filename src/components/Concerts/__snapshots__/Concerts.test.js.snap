// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Concerts should render correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
